src/BuildScriptGeneratorCli/Commands/BuildCommand.cs

```diff
@@ -266,7 +349,7 @@ internal override int Execute(IServiceProvider serviceProvider, IConsole console
                     return;
                 }
 
-                // Not using IConsole.WriteErrorLine intentionally, to keep the child's error stream intact
+                // Not using IConsole.WriteLine intentionally, to keep the child's error stream intact
```

Comment:
nit: I think we can revert the change in this comment, since we are still not using `WriteErrorLine` to preserve the error stream

Reply:
Though actually, this makes it seems as though `console.Error.WriteLine != console.WriteErrorLine` even though we are replacing the latter with the former later on in this code. Do you know how this affects the logging?

Reply:
Ok, yeah, so looks like WriteErrorLine is an Oryx defined method that prepends `Error:` to Error.WriteLine with `console.Error.WriteLine("Error: " + message)`:
https://github.com/microsoft/Oryx/blob/harrli/cliDotnetUpgrade/src/BuildScriptGeneratorCli/IConsoleExtensions.cs

Can we revert all of the changes from `console.WriteErrorLine -> console.Error.WriteLine` so we get the Error prepended again?